src/base/registration/CheckNameAvailability.svelte

```diff
@@ -1,6 +1,6 @@
 <script lang="ts">
```

Comment:
We name views that let you create a new resource `New`.

Reply:
Reverted in f26098eaa8f4187e99fd28e7be8db02f990d0d5c

Reply:
This was also my doing. I didn't know about this convention. My thinking behind it was, that the whole registration flow is broken up in multiple steps and I find it more descriptive if we'd name the steps for what they are supposed to accomplish. So:

`ValidateName.svelte` -> `CheckNameAvailability.svelte` -> `Register.svelte`

And then a separate view for viewing the registration: `View.svelte`.

This is also aligned with our routes, making it (at least from my perspective) easier to reason about.

```typescript
    {
      type: "registration";
      params: {
        activeView:
          | {
              type: "validateName";
            }
          | {
              type: "checkNameAvailability";
              params: { nameOrDomain: string; owner: string | null };
            }
          | {
              type: "register";
              params: { nameOrDomain: string; owner: string | null };
            }
          | {
              type: "view";
              params: { nameOrDomain: string; retry: boolean };
            };
      };
    }
```

Reply:
I think there should be a way to have both of those. Still I wouldn't want to have to look through all the components to figure out which one is the entry point for new registrations.

Reply:
Yeah, I think you're right about the entry point, that's something I'd also want to see from just looking at the code.

Reply:
Ok this commit reverts the renaming of the registration components 0669ddb43f1aa26dadb02faaa248f52a048908a6